raw-spans-grouper/raw-spans-grouper/src/test/resources/configs/raw-spans-grouper/application.conf

```diff
@@ -47,4 +47,10 @@ span.groupby.session.window.graceperiod.ms = 100
 
 dataflow.metriccollection.sampling.percent = 10.0
 
-trace.emit.punctuator.frequency = 15s
\ No newline at end of file
+trace.emit.punctuator.frequency = 15s
+
+peer.correlation = {
+  enabled.customers = ["*"]
```

Comment:
add a separate var: `enable.all.customers = true` or `false`

Reply:
This is the test `app.conf`, so added `*` in config. In the actual `app.conf` it is empty. What is the need of `enable.all.customers`, we can handle both using the same config.